system_baseline-manifest

```diff
@@ -48,21 +49,23 @@ services-drift:system_baseline/python:3.8=psycopg2-binary:2.9.3
 services-drift:system_baseline/python:3.8=pyparsing:3.0.9
 services-drift:system_baseline/python:3.8=pyrsistent:0.18.1
 services-drift:system_baseline/python:3.8=python-dateutil:2.8.2
-services-drift:system_baseline/python:3.8=pytz:2022.1
 services-drift:system_baseline/python:3.8=pytz-deprecation-shim:0.1.0.post0
 services-drift:system_baseline/python:3.8=pyyaml:6.0
-services-drift:system_baseline/python:3.8=redis:4.3.4
+services-drift:system_baseline/python:3.8=redis:4.4.0rc1
```

Comment:
Do we want to use release candidate version?

Reply:
We do not use redis at all in this project...

This won't make any difference

Reply:
+1 redis is being  pulled by other dependencies :(